openbb_terminal/alternative/covid/covid_controller.py

```diff
@@ -49,22 +49,22 @@ def __init__(self, queue: List[str] = None):
             }
             choices["ov"] = {
                 "--raw": {},
-                "--limit": {str(c): {} for c in range(1, 1000)},
+                "--limit": None,
                 "-l": "--limit",
             }
             choices["rates"] = {
                 "--raw": {},
-                "--limit": {str(c): {} for c in range(1, 1000)},
+                "--limit": None,
```

Comment:
I think these all need to be "--limit":{}

Reply:
@jmaslek whats the difference?

Reply:
actually I think it does not need to be that?  I seethe line above we do {} instead of None so I wasnt completely sure.